collector/src/main/java/com/bitmovin/analytics/BitmovinAnalytics.java

```diff
@@ -289,6 +297,21 @@ public void onAudioTrackChange() {
         data.setVideoTimeEnd(playerStateMachine.getVideoTimeEnd());
     }
 
+    @Override
+    public void onVideoStartFailed() {
+        String videoStartFailedReason = playerStateMachine.getVideoStartFailedReason() != null
+                ? playerStateMachine.getVideoStartFailedReason().toString().toLowerCase()
+                : VideoStartFailedReason.UNKNOWN.toString().toLowerCase();
+
+        EventData data = playerAdapter.createEventData();
+        data.setState(playerStateMachine.getCurrentState().toString().toLowerCase());
```

Comment:
do we want to set the state to `EXITBEFOREVIDEOSTART`?
i noticed in the web collector that this never happend

Reply:
I think web collector always sends some kind of state if failed attempt happens, and usually it's startup state. This field is nullable so we don't need to set it. Just wanted to make sure we are consistent with value of states.

Reply:
Ahh got it... it sets the State on enterState and then it contains the old state 👍